hibernate-core/src/test/java/org/hibernate/test/bytecode/enhancement/merge/MergeEnhancedDetachedOrphanRemovalTest.java

```diff
@@ -37,7 +38,10 @@ public void testMergeDetachedOrphanRemoval() {
 
 		doInHibernate( this::sessionFactory, session -> {
 			entity.setName( "updated" );
-			session.merge( entity );
+			Root entityMerged = (Root) session.merge( entity );
+			assertNotSame( entity, entityMerged );
+			assertNotSame( entity, entityMerged );
```

Comment:
why would you test this twice?

Reply:
It was a cut-and-paste error. I removed it in a later commit.